release/pkg/images/images.go

```diff
@@ -83,3 +91,276 @@ func CopyToDestination(sourceAuthConfig, releaseAuthConfig *docker.AuthConfigura
 
 	return nil
 }
+
+func GetSourceImageURI(r *releasetypes.ReleaseConfig, name, repoName string, tagOptions map[string]string) (string, string, error) {
+	var sourceImageUri string
+	sourcedFromBranch := r.BuildRepoBranchName
+	if r.DevRelease || r.ReleaseEnvironment == "development" {
+		latestTag := utils.GetLatestUploadDestination(r.BuildRepoBranchName)
+		if strings.Contains(name, "bottlerocket-bootstrap") {
```

Comment:
Similar thing here and in `GetReleaseImageURI`
I think we should avoid doing this and codify this information in the asset/image declaration

Reply:
I will work on making this as a configurable field, and have a default to just use git tag if not provided.